pkg/distro/fedora/distro.go

```diff
@@ -601,18 +640,11 @@ func newDistro(version int) distro.Distro {
 		&platform.X86{
 			BasePlatform: platform.BasePlatform{
 				FirmwarePackages: []string{
-					"microcode_ctl", // ??
-					"iwl1000-firmware",
-					"iwl100-firmware",
-					"iwl105-firmware",
-					"iwl135-firmware",
-					"iwl2000-firmware",
-					"iwl2030-firmware",
-					"iwl3160-firmware",
-					"iwl5000-firmware",
-					"iwl5150-firmware",
-					"iwl6000-firmware",
-					"iwl6050-firmware",
+					"biosdevname",
+					"grub2",
```

Comment:
I got tripped by `grub2` being still here. After looking around, it turns out that there is no `grub2` binary RPM. It is provided by the `grub2-pc` package.

```
[root@15aa9b685f65 /]# dnf provides grub2
Last metadata expiration check: 0:00:15 ago on Wed Aug 16 09:06:47 2023.
grub2-pc-1:2.06-89.fc38.x86_64 : Bootloader with support for Linux, Multiboot, and more
Repo        : fedora
Matched from:
Provide    : grub2 = 1:2.06-89.fc38

grub2-pc-1:2.06-95.fc38.x86_64 : Bootloader with support for Linux, Multiboot, and more
Repo        : updates
Matched from:
Provide    : grub2 = 1:2.06-95.fc38
```

So, it should not need to be listed here. I also checked other distros and we don't list `grub2` in any of them (not in platforms and also not in package sets).

`biosdevname` is also an interesting story... In other distros, we list it in the image package set. I would argue that it is not a firmware, so it probably should not be here. But I'm OK with keeping it as is for now...

Reply:
`biosdevname` should be added conditionally based on the option in the COI pipeline.

Reply:
I did the `biosdevname` conditional and dropped `grub2`.